src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -113,6 +113,15 @@ class SeekBar extends Slider {
    * @listens mousedown
    */
   handleMouseDown(event) {
+    // only allow left-click seeking
+    if (typeof event.button === 'undefined' && typeof event.buttons === 'undefined') {
+      // touch screen, not apply, do nothing
+    } else if (event.button === 0 && typeof event.buttons === 'undefined') {
+      // touch screen, safari on ios
+    } else if (event.button !== 0 || event.buttons !== 1) {
+      return;
+    }
```

Comment:
I find this series of `if` blocks confusing. Can it be simplified to:

```js
if ((event.button !== undefined && event.button !== 0) || (event.buttons !== undefined && event.buttons !== 1)) {
  return;
}
```

Does that not work?

Reply:
I don't know :/, what I did was trial by combat, didn't have device to test right now, think I will stick with what work, maybe change to that later if I have device to test

Reply:
Beside, I think if we do the current style, we have a chance to handle/explain what is going on with particular device, OS or stuff